ci/build_python.sh

```diff
@@ -9,11 +9,21 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+package_name="rmm"
+package_dir="python"
+
+version=$(rapids-generate-version)
+commit=$(git rev-parse HEAD)
+
+init_file="${package_dir}/${package_name}/__init__.py"
+sed -i "/__version__/ s/= .*/= ${version}/g" ${init_file}
+sed -i "/__git_commit__/ s/= .*/= \"${commit}\"/g" ${init_file}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏
```suggestion
sed -i "/^__version__/ s/= .*/= ${version}/g" ${init_file}
sed -i "/^__git_commit__/ s/= .*/= \"${commit}\"/g" ${init_file}
```

Let's anchor this to the beginning of the line so that `# __version__ = ...` is not rewritten.